components/micro-gateway-core/src/main/ballerina/src/gateway/utils/utils.bal

```diff
@@ -40,6 +40,8 @@ map<FilterConfiguration?> filterConfigAnnotationMap = {};
 map<http:InboundAuthHandler> authHandlersMap = {}; //all handlers except for jwt handlers
 http:InboundAuthHandler[] jwtHandlers = [];//all jwt issuer handlers
 string authHeaderFromConfig = getConfigValue(AUTH_CONF_INSTANCE_ID, AUTH_HEADER_NAME, DEFAULT_AUTH_HEADER_NAME);
+map<string> openAPIs = {};
```

Comment:
I think we can save the open APIs at the java side

Reply:
fixed